src/Spatial/Euclidean/Plane.cs

```diff
@@ -74,6 +77,30 @@ public Plane(Point3D rootPoint, UnitVector3D normal)
         {
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Plane"/> struct.
+        /// Constructs a Plane from the given points.
+        /// </summary>
+        /// <param name="points">The points</param>
+        /// <returns>Fitted <see cref="Plane"/></returns>
+        /// <remarks>this method uses SVD to fit the plane.</remarks>
+        public static Plane CreateFittedPlaneFrom(IEnumerable<Point3D> points)
+        {
+            var c = Point3D.Centroid(points);
+
+            var A = CreateMatrix.DenseOfRowVectors(
+                points.Select(p => p - c)
+                    .Select(p => CreateVector.DenseOfArray(new double[] { p.X, p.Y, p.Z })));
+
+            var svd = A.Svd(true);
+            var matV = svd.VT.Transpose();
+            var theIndex = svd.S.Count-1; // in this case, theIndex = 2.
```

Comment:
I think you can entirely skip this variable and use `var normal = UnitVector3D.OfVector(matV.Column(matV.Cols - 1));`. This better captures semantically the intention, that we need the last eigenvector of V.